client/src/featureform/register.py

```diff
@@ -2810,9 +2810,9 @@ def register_cassandra(
     def register_dynamodb(
         self,
         name: str,
-        access_key: str,
-        secret_key: str,
+        credentials: AWSCredentials,
         region: str,
+        import_from_s3: bool = False,
```

Comment:
should this be `use_import_from_s3`? I know it's longer

Reply:
We could. Are you suggesting the change to make the param more boolean sounding? If so, I think we should go with `should_import_from_s3`, yeah?

Reply:
should this always be done by default?

Reply:
in fact when would you even want it to be off?